src/script/lua_api/l_object.cpp

```diff
@@ -2363,7 +2363,14 @@ int ObjectRef::l_set_lighting(lua_State *L)
 			lighting.exposure.center_weight_power = getfloatfield_default(L, -1, "center_weight_power", lighting.exposure.center_weight_power);
 		}
 		lua_pop(L, 1); // exposure
-	}
+
+		lua_getfield(L, 2, "volumetric_light");
+		if (lua_istable(L, -1)) {
```

Comment:
do you forsee any other parameters or why a table?

Reply:
Yeah, like shadows, we might add new parameters in the future, such as custom tints as mentioned above.